src/java/org/apache/nutch/parse/ParseSegment.java

```diff
@@ -43,20 +50,16 @@
 import java.util.Map.Entry;
 
 /* Parse content in a segment. */
-public class ParseSegment extends NutchTool implements Tool,
-    Mapper<WritableComparable<?>, Content, Text, ParseImpl>,
-    Reducer<Text, Writable, Text, Writable> {
+public class ParseSegment extends NutchTool implements Tool {
 
   private static final Logger LOG = LoggerFactory
       .getLogger(MethodHandles.lookup().lookupClass());
 
   public static final String SKIP_TRUNCATED = "parser.skip.truncated";
 
-  private ScoringFilters scfilters;
+  private static ScoringFilters scfilters;
 
-  private ParseUtil parseUtil;
-
-  private boolean skipTruncated;
+  private static  boolean skipTruncated;
```

Comment:
See comments in Generator etc. regarding static variables shared between mapper and reducer classes.